cmd/collector/main_test.go

```diff
@@ -28,6 +28,23 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+func TestGetDefaultCollectorConfigPathENV(t *testing.T) {
+	os.Setenv(configPathENV, "./fake/path/config.yaml")
```

Comment:
nit: This string literal `"./fake/path/config.yaml"` is used twice. We can just store it in an variable so it's harder to mess up if we ever change this.

Reply:
Good call, I went in and changed it